### DISCUSSION TITLE
标题超20字断行会跑到下一页，这个问题该怎么解决

### DISCUSSION THREAD

**Discussion by huiling-liao** [sikouhjw/gdutthesis#48] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/56637074/158726044-ed892e93-61de-437d-a25e-13f92d4ae271.png)

**Comment by sikouhjw** [comment, marked as answer]
https://github.com/sikouhjw/gdutthesis/issues/19

**Reply by huiling-liao** [reply]
谢谢啦
